p2p/switch.go

```diff
@@ -94,8 +95,11 @@ type Switch struct {
 
 // NetAddress returns the address the switch is listening on.
```

Comment:
```suggestion
// NetAddress returns the first address the switch is listening, or nil if no addresses are found.
```

Reply:
This will be removed when we implement the router anyway, but sure.